### DISCUSSION TITLE
What postgres version do you use currently?

### DISCUSSION THREAD

**Discussion by sjib** [QGEP/QGEP#783] [Polls]
For defining minimal / maximal requirements we would like to know what [Postgres version](https://www.postgresql.org/) you currently use?
`SELECT version();`

Please also fill out the [question about your postgis version](https://github.com/QGEP/QGEP/discussions/782) used.

**Comment by cymed** [comment]
Do you know by chance if the java-problems with scram-sha-256 authentication have been fixed?

**Reply by sjib** [reply]
@cymed Here is the discussion on this topic and all workarounds: https://github.com/claeis/ili2db/issues/448

**Reply by ponceta** [reply]
This is really a potential issue. I hope they have something in mind to fix it quickly otherwise it's bye bye to Ili2db...
